### PR TITLE
docs: add code samples for `read_gbq_function` using community UDFs

### PR DESCRIPTION
**Pull request by ashleyxuu**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes b/307559272 🦕

### CODE REVIEW

**Comment by ashleyxuu** [comment]
You can checkout the staging here: https://cloud.devsite.corp.google.com/python/docs/reference/bigframes/latest/bigframes.pandas.html#bigframes_pandas_read_gbq_function

**Review by shobsi — APPROVED** [review]

File: bigframes/session/__init__.py [hunk 1]
```diff
@@ -1456,15 +1456,37 @@ def read_gbq_function(
             The return type of the function must be explicitly specified in the
             function's original definition even if not otherwise required.
 
+        BigQuery UDFs has many public functions under the ``bqutil`` project on publicly shared datasets
```
Comment (tswast) [inline bigframes/session/__init__.py:1459]: I'm finding this sentence difficult to understand. Perhaps, something like this?

```suggestion
        BigQuery Utils provides many public functions under the ``bqutil`` GCP project
```

File: bigframes/session/__init__.py [hunk 2]
```diff
@@ -1456,15 +1456,37 @@ def read_gbq_function(
...
+        (See: https://github.com/GoogleCloudPlatform/bigquery-utils/tree/master/udfs#using-the-udfs).
+        You can checkout Community UDFs to use community-contributed functions.
+        (See: https://github.com/GoogleCloudPlatform/bigquery-utils/tree/master/udfs/community#community-udfs).
+
         **Examples:**
 
+        Using the ``cw_lower_case_ascii_only`` function from Community UDFs.
```
Comment (tswast) [inline bigframes/session/__init__.py:1466]: Nit: verb should be active, telling the user what to do.

```suggestion
        Use the ``cw_lower_case_ascii_only`` function from Community UDFs.
```

**Review by tswast — COMMENTED** [review]
Love it! A couple of suggestions to improve the text.

**Pull request merged**